lib/unzipGeoStream.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var path = require('path');
+var mkdirp = require('mkdirp');
+var unzip = require('unzip');
+var rmrf = require('rimraf');
+
+
+function unzipGeoStream(zip, stream, counter, geoCb, finalCb){
+  var dirname = path.join(path.dirname(zip), Math.round(Math.random()*1e15).toString(), path.basename(zip, '.zip'));
```

Comment:
What's going on here?  Are you generating a random directory name?  I assume so multiple runs of the loader won't unzip the file to the same destination?  A comment for the intent would be helpful, perhaps with an example of what the directory name should look like.